runtime/doc/deprecated.txt

```diff
@@ -116,9 +116,19 @@ internally and are no longer exposed as part of the API. Instead, use
 - *vim.lsp.diagnostic.set_underline()*
 - *vim.lsp.diagnostic.set_virtual_text()*
 
+DIAGNOSTIC API
 Configuring |diagnostic-signs| with |:sign-define| or |sign_define()| is no
 longer supported. Use the "signs" key of |vim.diagnostic.config()| instead.
 
+- *vim.diagnostic.goto_next()*	Use |vim.diagnostic.goto()| instead.
+- *vim.diagnostic.goto_prev()*	Use |vim.diagnostic.goto()| instead.
```

Comment:
Please consider suggesting more explicit alternative, e.g. `vim.diagnostic.goto({ count = -1 })` along with the helptag.

Reply:
I want to leave the link to `vim.diagnostic.jump()` in place. I changed it to the following:

```
- *vim.diagnostic.goto_next()*	Use |vim.diagnostic.jump()| with `{count = 1}` instead.
- *vim.diagnostic.goto_prev()*	Use |vim.diagnostic.jump()| with `{count = -1}` instead.
```